### PR TITLE
[REF] gamification: review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2938963

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @stefanorigano 👋 
This PR is ready for review, could you take a look at it ?

**Comment by Brieuc-brd** [comment]
@stefanorigano Little reminder ;)

**Review by stefanorigano — APPROVED** [review]
LGTM 👍

**Comment by stefanorigano** [comment]
@adr-odoo , ready for tech. review 👍

File: addons/gamification/views/gamification_goal_views.xml [hunk 1]
```diff
@@ -169,10 +169,10 @@
                         <div t-attf-class="oe_kanban_card oe_kanban_global_click o_kanban_gamification #{record.end_date.raw_value &lt; record.last_update.raw_value and record.state.raw_value == 'failed' ? 'oe_kanban_color_2' : ''} #{record.end_date.raw_value &lt; record.last_update.raw_value and record.state.raw_value == 'reached' ? 'oe_kanban_color_5' : ''}">
                             <div class="o_kanban_content">
```
Comment (adr-odoo) [inline addons/gamification/views/gamification_goal_views.xml:170]: Could we add text-center on this node and remove all text-center from its childs?

**Review by adr-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Pull request closed without merging**